web-api/src/models.rs

```diff
@@ -77,13 +99,15 @@ macro_rules! string_wrapper {
                 type Error = $error;
 
                 fn try_from(mut value: String) -> Result<Self, Self::Error> {
+                    use ::std::ops::RangeInclusive;
+
```

Comment:
:art: since the macro is not exported, we can just use the module import declared above and don't have to import it again
```suggestion
```